domains/zenith.json

```diff
@@ -0,0 +1,10 @@
+{
+    "description": "Zenith's main domain",
+    "owner": {
+        "username": "ZenithGuy1",
+        "email": "matheusbarcelosas@gmail.com"
+    },
+    "record": {
+        "TXT": "By Zenith"

```

Comment:
TXT records alone does nothing to your website.

Reply:
I know it does nothing. I dont want it to do anything as well.